pagopa/io-app
# chore(IT Wallet): [SIW-1536] Add skeumorphic card in MDL details screen

## Short description
This PR adds the skeumorphic card in the credential details screen.
The skeumorphic card is rendered only if the credential supports it (at the time of opening this PR only the Driving License is supported).

## List of changes proposed in this pull request
- Updated `io-app-design-system` to `1.46.0`
- Added background as a fall back while the `ItwSkeumorphicCard`  background is being loaded
- Added the `ItwSkeumorphicCard` component in the details screen for credentials which sopport it

## How to test
Open the details screen of your MDL credential, check that the card is rendered correctly and you can flip it.

## Preview

https://github.com/user-attachments/assets/5bbce011-919c-4c90-8701-c736bbce70de
